example-tree.json

```diff
@@ -1,65 +1,74 @@
 [
     {
-        "id": 0,
-        "title": "Self Improvement Skill Tree",
-        "category": "root",
-        "link": "https://www.youtube.com/watch?v=PYaixyrzDOk",
-        "type": "item",
-        "requires": [],
-        "children": [
-            1,
-            2
-        ]
-    },
-    {
-        "id": 1,
-        "iconName": "mindfulness",
-        "title": "Mindfulness I",
-        "level": 1,
-        "goal": "Mindfulness 2mins",
-        "frequency": "DAILY",
-        "timelimit": "1x Week",
-        "xp": 50,
+        "title": "babble",
+        "level": 2,
+        "goal": [
+            "annihalate infants",
```

Comment:
change that to something else like "example goal" 😆

Reply:
Sorry about that one, was listening to infant annihilator and couldn't come up with a better example. Will do